include/cupla/namespace.hpp

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2019 Andrea Bocci
+ *
+ * This file is part of cupla.
+ *
+ * cupla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * cupla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with cupla.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+#pragma once
+
+#if CUPLA_STREAM_ASYNC_ENABLED
+
+#ifdef ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLED
```

Comment:
@psychocoderHPC regarding this block: are accelerators enabled exclusively in cupla?

Reply:
@ax3l I think I've asked @psychocoderHPC the same not long ago. iirc, yes, cupla only supports a single accelerator per configuration (unlike alpaka).

Reply:
Yes currently the accelerator is mostly exclusive. Cupla is designed as replacement for for cuda where you also only have one accelerator.
There is a special mode in cupla where you can have two accelerator but only to help during the transition from pure cuda to cupla.
In the furture I like to releax the condition and allow multiple accelerators.
This will be part of future refactorings.